### ISSUE TITLE
Authentication/Check not providing verification code text.

### ISSUE THREAD

**Issue by togacaptain** [hatamiarash7/Clubhouse-API#3]
I was testing this out and found a discrepancy between your example and the actual.

When running the Authentication/Check call I received the following response body.

{
    "success": true,
    "is_blocked": false,
    "error_message": null,
    "send_rc_token": false
}

Notice the "send_rc_token":false that isn't in your example response. Additionally I'm not getting the expected SMS text with the verification code. 

However if I go to the app on my phone and logout/login I can trigger the SMS text and then take that verification code and plug it in to your Authentication/Complete call and it works fine.

Any idea what has changed on the Authentication/Check call that keeping the SMS text from being sent? It seems like once authenticated everything else still works as expected.

**Comment by nazhmik** [comment]
Hi, @togacaptain. Did you find a solution to the problem?

**Comment by togacaptain** [comment]
No. I found a couple of other projects on here that said they had it working, but when testing them out I couldn't make them work. At this point I have put this aside until a definitive solution can be found or Clubhouse releases a public development api.

**Comment by togacaptain** [comment]
> Hi, @togacaptain. Did you find a solution to the problem?

No @nazhmik. I found a couple of other projects on here that said they had it working, but when testing them out I couldn't make them work. At this point I have put this aside until a definitive solution can be found or Clubhouse releases a public development api.
